feat: Add Location Page

suinj8 opened the pull request:
1.  소개 -> 센터위치 페이지 구현하였습니다.
2.  Kakao Map API 이용하였습니다.
3. 기존 디자인 내용에서 아래부분에 상세내용 추가하였습니다.
   (스크린샷 첨부합니다)
4. 3번 내용에 Slide 컴포넌트와 스크롤의 위치를 이용하여
   특정 위치가 되면 slide 컴포넌트가 등장하도록 하였습니다.

![image](https://user-images.githubusercontent.com/90179555/166241950-b5bee48f-1097-4cbf-b7a5-d3f47c1eb5bc.png)

suinj8 committed:
feat: Add Location Page
소개 - 센터위치 페이지 구현하였습니다.
1. Kakao Map API이용하여 공대 7호관을 나타내게 하였습니다.
2. 기존 디자인에서 서울대 홈페이지를 참고하여 유사하게 꾸며보았습니다.
3. 아이콘을 이용하여 직관적이게 만들었습니다.

suinj8 committed:
feat: Update Location Page

1. Slide 컴포넌트를 이용하여 동적으로 보일 수 있게 수정하였습니다.
2. 스크롤 위치를 파악하여 위치에 따라 위 Slide 컴포넌트가 나오도록
   수정하였습니다.

LimEunSang commented on src/pages/introduction/Location.js:
```diff
@@ -0,0 +1,273 @@
+ */
+
+const Location = () => {
+  // Kakao Map API 사용부분
```
!! 여기 주석 부분

LimEunSang commented on src/pages/introduction/Location.js:
```diff
@@ -0,0 +1,273 @@
+      setContentAppear(true);
+    }
+  };
+
```
!! 여기에 있어야 합니다!

LimEunSang commented on src/App.js:
```diff
@@ -2,6 +2,7 @@ import { createContext } from "react";
 import Home from "./pages/Home";
+import Location from "./pages/introduction/Location";
 
 export const ArticleContext = createContext();
```
지금 발견했는데 이름 바꿔야겠네요.
develop 브랜치에서 바꿉시다.

LimEunSang commented on src/pages/introduction/Location.js:
```diff
@@ -0,0 +1,273 @@
+  const [titleAppear, setTitleAppear] = useState(false);
+  const [contentAppear, setContentAppear] = useState(false);
+
+  const listener = (e) => {
```
무엇을 위한 기능인지 설명이 필요해 보입니다!

LimEunSang reviewed (commented):
노력의 흔적이 여기까지 느껴집니다
수고했습니다!!!

LimEunSang reviewed (commented):
.

suinj8 committed:
refactor: Update Location Page

지적해주신 code review 수정
1. 주석 위치 수정
2. 주석 추가

LimEunSang commented:
> .

suinj8 committed:
feat: Update Location Page

1. 스크롤 위치의 비율을 구해서 특정 비율을 넘으면
   컴포넌트가 등장했다가 사라지도록 수정하였습니다.
2. 약간의 코드수정으로 주석도 조금 수정하였습니다.

baaaam771 commented on src/pages/introduction/Location.js:
```diff
@@ -0,0 +1,286 @@
+          />
+
+          <EmailIcon />
+          <Typography>&nbsp;jbnuAI@jbnu.ac.kr</Typography>
```
jbnuAI@jbnu.ac.kr 이메일은 임의로 작성해주신거죠?

suinj8 replied:
맞습니다~

baaaam771 reviewed (commented):
카카오 지도 api 잘 적용해주셨네요 고생하셨어요
스크롤을 내리면 slide 컴포넌트 보이는 부분을 사용자가 인지할 수 있게 해줘야할지 고민해봐도 좋을것 같아요

baaaam771 reviewed (approved):
LGTM 고생하셨습니다!

LimEunSang reviewed (approved):
변경사항 확인했습니다
LGTM

euny0ung commented:
LGTM 수고하셨습니다!

suinj8 committed:
Merge branch 'develop' into feature/dev_location_page
